shopify_app.gemspec

```diff
@@ -14,6 +14,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('rails', '> 5.2.1')
   s.add_runtime_dependency('shopify_api', '~> 8.0')
   s.add_runtime_dependency('omniauth-shopify-oauth2', '~> 2.2.0')
+  s.add_runtime_dependency('dotenv-rails', '~> 2.7.5')
```

Comment:
Why was this added a runtime dependency? This forces all apps to add dotenv, which not really needed.